week04/style.css

```
@@ -0,0 +1,96 @@
+
+
+html,body{
```

body 태그에서 color를 설정하면, 자식 요소에 일괄 적용됩니다. 자식 요소 각각에 대하여 중복으로 코드를 작성할 필요가 없습니다.
```suggestion
body {
   color: white;
```